wiki/Disambiguation/de.md

```diff
@@ -1,15 +1,22 @@
----
-outdated: true
-outdated_since: 8043b3a603ef5a8bb7ab9f982fceed3825cb249f
----
-
 # Begriffserklärung
 
 Die Begriffserklärung ist ein Prozess der Lösung von Konflikten die entstehen können, wenn der Titel eines Artikels nicht eindeutig ist. Meistens ist dies der Fall dadurch, dass er sich auf mehr als nur ein vom osu! wiki behandeltes Thema bezieht, entweder als Haupt- oder Unterthema.
 
+Ein normaler Artikel, mit einem ähnlichen Titel, muss die Kopfnote ["Für andere Bedeutungen"](/wiki/Article_styling_criteria/Formatting#for-other-uses) mit einem Link zur Seite für die Begriffserklärung haben.
+
 ## Auflistung uneindeutiger Artikeltitel
 
 - [Bancho](Bancho)
 - [Easy](Easy)
 - [EZ](EZ)
 - [Mod](Mod)
+- [Combo](Combo)
+- [Fail](Fail)
+- [NC](NC)
+- [NM](NM)
+- [Offset](Offset)
+- [osu!](osu!)
+- [Ouendan](Ouendan)
+- [Rank](Rank)
+- [Star](Star)
+- [Version](Version)
```

Comment:
The english list is sorted alphabetically, I believe this one should be the same.

Reply:
Yeah, seems plausible. I just copy-pasted the missing ones 🤷